iphone/cli/commands/_buildModule.js

```diff
@@ -476,6 +483,25 @@ iOSModuleBuilder.prototype.buildModule = function buildModule(next) {
 			args.push('SUPPORTS_MACCATALYST=YES');
 		}
 
+		if (target === 'iphonesimulator') {
+			// Search for third party ramewrok included in module. If found any exclude arm64. 
+			// Assumption is that simulator arm64  architecture can only be supported via .xcframework.
+			const frameworkPattern = /([^/]+)\.framework$/;
+			let frameworksPath = path.join(this.projectDir, 'platform');
+			let legacyFrameworks = new Set();
+
+			fs.readdirSync(frameworksPath).forEach(function (filename) {
+						let possibleFrameworkPath = path.join(frameworksPath, filename);
+						if (frameworkPattern.test(possibleFrameworkPath)) {
+							legacyFrameworks.add(filename);
+						}
```

Comment:
<!--
  1 failure:  iphone/cli/comman...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/commands/_buildModule.js;
  Line: 497;
-->
- :no_entry_sign: iphone/cli/commands/_buildModule.js line 497 – Expected indentation of 4 tabs but found 6. (indent)